ecf/scripts/gfs/atmos/post_processing/grib2_wafs/jgfs_atmos_wafs_grib2_0p25.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:30:00
-#PBS -l select=1:mpiprocs=11:ompthreads=1:ncpus=11:mem=80GB
+#PBS -l select=1:mpiprocs=39:ompthreads=1:ncpus=39:mem=200GB
```

Comment:
I am sure NCO will ask why we are increasing this and why it is not reflected in the Release notes.
@YaliMao-NOAA Can you please justify these increases?

Reply:
@aerorahul Xiaoxue, who is in charge of this implementation, pointed out the change that I missed in the previous PR. Should we still include in the release notes?

Reply:
It is a good idea to make a note in the release notes as we document all changes from the previous release.

Reply:
@KateFriedman-NOAA Sorry I didn't think about the whole situation throughly. I made another PR #2050 for the release note update.